doc/Makefile

```diff
@@ -0,0 +1,49 @@
+# Makefile for RISC-V Doc Template
+#
+# This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+# International License. To view a copy of this license, visit
+# http://creativecommons.org/licenses/by-sa/4.0/ or send a letter to
+# Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
+#
+# SPDX-License-Identifier: CC-BY-SA-4.0
+#
+# Description:
+# 
+# This Makefile is designed to automate the process of building and packaging 
+# the Doc Template for RISC-V Extensions.
+
+DATE ?= $(shell date +%Y-%m-%d)
+VERSION ?= v0.0.0
```

Comment:
Fill the current version, and I would prefer use 0.13 rather than 1.0 right now, even we don't plan to did any incompatible change, and then bump it to 1.0 until we froze this :)

Reply:
Since the Github workflow will be punching snapshots into [release](https://github.com/riscv-non-isa/rvv-intrinsic-doc/releases) with the title of "draft release", I say we can alter change this by the time we are freezing them for v1.0 or tagging for another compiler release.

Reply:
Tagging the spec as v0.13.0-draft